### DISCUSSION TITLE
Mouse hover stats

### DISCUSSION THREAD

**Discussion by MrGys** [seerge/g-helper#125] [Ideas]
Is it possible when hovering with mouse in tray area for app to display CPU temp, discharge ratio or performance modes etc.

**Comment by seerge** [comment]
[GHelper.zip](https://github.com/seerge/g-helper/files/11042055/GHelper.zip)
Like this one ? :) (i was also thinking about showing at least a discharge rate in a tooltip)

**Reply by MrGys** [reply]
Wow that was fast, great job! :)

**Reply by seerge** [reply]
I have also pushed it (by just reuploading) to v0.36.
